deduplication/models.py

```diff
@@ -0,0 +1,137 @@
+import pickle
+from django.db import models
+
+import logging
+
+from core_server.base_models import BaseModel
+from core.models import Lead, Project
+
+
+logger = logging.getLogger(__name__)
+
+
+class LSHIndex(BaseModel):
+    """
+    Uses datasketch LSH Index
+    """
+
+    THRESHOLD = 0.55  # The value comes from experiments conducted on test CORE dataset
```

Comment:
Pass this from the environment.

Reply:
I think not. This is something that's not changed and it needs to be very closely tied up with the model. So Let's keep it this way.